blagues.json

```
@@ -6038,7 +6038,391 @@
   {
     "id": 1007,
     "type": "global",
-    "joke": "Chuck Norris peut mettre un headshot ",
+    "joke": "Chuck Norris peut mettre un headshot.",
     "answer": "Au cavalier sans tête."
+  },
+  {
+    "id": 1008,
+    "type": "dev",
+    "joke": "Quel est le comble pour la mémoire vive ?",
+    "answer": "C'est qu'elle RAM."
+  },
+  {
+    "id": 1009,
+    "type": "dark",
+    "joke": "Savez-vous pourquoi le petit Grégory n'était jamais en retard ?",
+    "answer": "Parcequ'il était dans l'étang !"
+  },
+  {
+    "id": 1010,
+    "type": "global",
+    "joke": "Quel est le berger le plus connu ?",
+    "answer": "Le seigneur des agneaux."
+  },
+  {
+    "id": 1011,
+    "type": "global",
+    "joke": "Qu'est-ce qui fait rire un cuisinier ?",
+    "answer": "Une farce"
+  },
+  {
+    "id": 1012,
+    "type": "beauf",
+    "joke": "Quel est l'application musique à l'heure ?",
+    "answer": "Deezer !"
+  },
+  {
+    "id": 1013,
+    "type": "dark",
+    "joke": "Le métier de croque-mort n'a aucun avenir.",
+    "answer": "Les clients ne sont pas fidèles."
+  },
+  {
+    "id": 1014,
+    "type": "global",
+    "joke": "Pourquoi les marins ne peuvent t'il pas écrire ?",
+    "answer": "Parce que ils ont jeté l'ancre."
+  },
+  {
+    "id": 1015,
+    "type": "dark",
+    "joke": "Les humains c'est comme le chocolat.",
+    "answer": "Ceux qui préfèrent le noir ont mauvais goût."
+  },
+  {
+    "id": 1016,
+    "type": "dark",
+    "joke": "Les humains c'est comme les pensées.",
+    "answer": "On les aimes pas en noir."
+  },
+  {
+    "id": 1017,
+    "type": "global",
+    "joke": "Quelles sont les lettres les moins visibles ?",
+    "answer": "F.A.C"
+  },
+  {
+    "id": 1018,
+    "type": "dark",
+    "joke": "Pourquoi la femme d'Hitler l'a-t'elle quitté ?",
+    "answer": "Elle a vu la facture de gaz."
+  },
+  {
+    "id": 1019,
+    "type": "dark",
+    "joke": "Les blagues noires c'est comme les enfants qui ont le cancer,",
+    "answer": "Ça ne vieillit jamais..."
+  },
+  {
+    "id": 1020,
+    "type": "dark",
+    "joke": "Pourquoi une imprimante arabe ne peux pas fonctionner ?",
+    "answer": "Car elle n'a pas de papiers."
+  },
+  {
+    "id": 1021,
+    "type": "dark",
+    "joke": "La vie, c'est comme un 100 mètres.",
+    "answer": "Les obèses arrivent pas jusqu'au bout."
+  },
+  {
+    "id": 1022,
+    "type": "global",
+    "joke": "Comment appelle-t-on un avion qui rebondit ? ",
+    "answer": "Un boing !"
+  },
+  {
+    "id": 1023,
+    "type": "global",
+    "joke": "Pourquoi certaines personnes jettent-elles leur ordinateur à la mer ?",
+    "answer": "Pour surfer sur le net."
+  },
```

```suggestion
  {
    "id": 1023,
    "type": "global",
    "joke": "Pourquoi certaines personnes jettent-elles leurs ordinateurs à la mer ?",
    "answer": "Pour surfer sur le net."
  },
```
J'ai un petit doute mais pour moi, c'est ça